integ/src/nodegroup_provider.rs

```diff
@@ -465,7 +466,7 @@ fn userdata(
         _ => return Err(ProviderError::new_with_context("Invalid dns ip")),
     };
 
-    Ok(base64::encode(format!(
+    Ok(general_purpose::STANDARD.encode(format!(
```

Comment:
nit: It would be nice to `use base64::engine::general_purpose::STANDARD as base64_` or similar to clarify this line.

Reply:
Yeah, good call. I don't love the API on this, so at least aliasing that would make it a little more clear what is being done there.

Tested the change with the Rust playground again to make sure there are no differences.

Since there are a few approvals here, going to merge once tests pass to avoid taking everyone's time on something minor. But if anyone has any concerns about this last update, please let me know and I will be happy to revert or update to address those concerns.